plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors.
```

Comment:
Oh, this is a existing file, you should not change the boilerplate

Reply:
is it a community convention?

Reply:
I think so, otherwise you should not have seen files in repo with boilerplate of 2016 or earlier :)